_clanky/neziskovky-automatizace-dashboardu.md

```diff
@@ -0,0 +1,97 @@
+
+Když teď skript spustíte (Tools>Build v editoru Sublime), automaticky se vám vaše data v google sheetu aktualizují. Spouštět skript jednoduchým stlačením tlačítka můžete provádět pravidelně dle potřeby. V případě, že je nutná velmi častá aktualizace, např. každý den, je možné nastavit vaše pc pomocí Task Scheduler na [automatické spouštění skriptu][PLANOVAC].
+
+Po úspěšném nastavení skriptu se nám tak už všechna data dostávají automaticky do našeho Google Sheetu. V případě neziskových organizací jsou data úplná a srozumitelná - pro to aby jsme je dokázali číst nepotřebujeme žádnou dokumentaci nebo manuál. Může se však stát, že i po nahrání do sheetu bude třeba data ještě upravovat tak, aby byla srozumitelná a prezentovatelná pro veřejnost. Abyste data nemuseli po každém načtení znova a znova upravovat a čistit, je dobré využít možnosti [makro][MAKRO], kterou google sheet nabízí. Jednoduše vám to umožní automatizovat čištění a úpravy, které jste už jednou dělali.
+
+Teď, když máte tuto práci za sebou a data jsou očištěna, je potřeba je ještě vizualizovat. Pro tento účel je možné využít širokou paletu nástrojů zdarma jako například [Tableau Public][TABLEAU] nebo [PowerBI][POWERBI]. My jsme pro účely článku zvolili nástroj [Google Data Studio][DATASTUDIO]. Obdobně jako jiné nástroje je zdarma a nabízí široké možnosti zpracování a vizualizace dat. Výhodou je také velmi jednoduché a přímočaré napojení dashboardu na google sheets a přívětivé uživatelské rozhraní. Navíc s vytvořením Google účtu se vám automaticky vytvořil i účet Google Data Studio a nemusíte řešit další registrace. Na konci dne tak můžete mít podobný, automaticky aktualizovaný dashboard nad daty z [NKOD][NKOD].
+
+<div class="embed-responsive embed-responsive-16by9">
+<iframe class="embed-responsive-item" src="https://datastudio.google.com/reporting/ca645c4e-c91c-4b40-8f09-98bfccb72d2d" frameborder="0" allowFullScreen></iframe>
```

Comment:
Tadyto mi nefunguje. Možná to je ale testováním u sebe a na webu to půjde?
![image](https://user-images.githubusercontent.com/718827/106704443-71bf9400-65ec-11eb-8116-91aab6647c03.png)

Technicky u iframů si myslím že je problém z hlediska GDPR a Cookies - my je nepoužíváme, ale ten iframe bych řekl že ano. V tu chvíli bychom potřebovali celou tu šaškárnu kolem souhlasů atd. a nějak to komunikovat googlu. Byl bych pro na to z tohoto důvodu dávat z článku jen odkaz a neembedovat to... ale můžeme o tom diskutovat.

Reply:
@jakubklimek ahoj, tak som všetky pripomienky zapracoval a pushol. Čo sa týka toho iframe tak máš pravdu a navyše je to problém aj na mobiloch často takže bude lepšie keď tam bude link. Díky a krásny večer.